spec/models/step_spec.rb

```diff
@@ -40,7 +40,7 @@
       expect(step_item.errors).to have_key(:logic)
     end
   end
-  describe 'broken_links' do
+  describe '.broken_links' do
```

Comment:
Not really needed, but if you do want to edit the test name, can you split it out because it's not related to the other changes in this commit.